backend/spec/requests/answers_survey_request_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'AnswersSurveyController', type: :request do
+  let(:user_student) { create(:user_student) }
+  let(:survey) { create(:survey_with_2_questions) }
+  let(:answers_survey_params) { { survey_id: survey.id } }
+
+  describe '#create' do
+    subject(:call) { post answers_surveys_path, params: answers_survey_params, headers: auth_headers(user: user_student) }
+
+    it 'response status' do
+      call
+      expect(response).to have_http_status :found
```

Comment:
:found status is returned, but in my mind should be :created, does it make sense?

I am not familiar with this kind of test so I have no strong opinion about this